save.py

```diff
@@ -10,7 +10,8 @@
 
 import aiohttp_jinja2
 
-from nameinternal import get_card, get_card_metadata
+from score.ScoreBonus import *
```

Comment:
In general, I really dislike star imports. I would prefer to refer to them with the module, e.g. `from score import ScoreBonus as _s`, and then do `_s.get_floors_climbed_bonus` etc.